public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
정말 nit 하긴 한데 한글 서비스이니 ko로 바꾸는 것은 어떤가요? 
```suggestion
<html lang="ko">
```

Reply:
오 미처 신경을 못썻던 부분입니다! 감사합니다